### PR TITLE
maint: Drop 3.7 support

### LINKED ISSUE #430: Remove Python 3.7 support
Following https://github.com/pyansys/pyansys/pull/228 you can now stop supporting Python 3.7 officially. Not required anymore by the metapackage. Assigning to @AlejandroFernandezLuces

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
Removes support for python 3.7

Closes #430

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -7,7 +7,6 @@ isolated_build_env = build
 
 [testenv]
 basepython =
-    py37: python3.7
```
Comment (RobPasMue) [inline tox.ini:10]: Remove it also from line 3 in this same file

File: .github/workflows/ci_cd.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
```
Comment (RobPasMue) [inline .github/workflows/ci_cd.yml:64]: Just realized, macOS wheels are missing. Generate them only on release. See https://github.com/pyansys/pygeometry/blob/0108428c4d546fb47333e0c0b4bea1f4e41ce7f4/.github/workflows/ci_cd.yml#L48-L61

File: .github/workflows/ci_cd.yml [hunk 3]
```diff
@@ -81,7 +81,7 @@ jobs:
     strategy:
       fail-fast: false
```
Comment (RobPasMue) [inline .github/workflows/ci_cd.yml:82]: And delete this stage. Better in one single step

File: pyproject.toml [hunk 4]
```diff
@@ -62,7 +62,7 @@ name = "ansys.meshing.prime"
 
 [tool.black]
 line-length = 100
-target-version = ['py37', 'py38', 'py39', 'py310']
+target-version = ['py38', 'py39', 'py310']
```
Comment (RobPasMue) [inline pyproject.toml:65]: Missing 3.11

File: .github/workflows/ci_cd.yml (hunk 2 above)
Comment (AlejandroFernandezLuces) [reply .github/workflows/ci_cd.yml]: Not sure if I did what you wanted, can you check it?

Comment (RobPasMue) [reply .github/workflows/ci_cd.yml]: Yep, that's it. You were jst missing to delete the ``macos_build`` stage since it is no longer necessary

**Review by RobPasMue — DISMISSED** [review]

File: pyproject.toml [hunk 5]
```diff
@@ -7,15 +7,14 @@ name = "ansys-meshing-prime"
 version = "0.4.0.dev7"
 description = "PyPrimeMesh provides a python client to Ansys Prime Server. Ansys Prime Server delivers core Ansys meshing technology."
 readme = "README.md"
-requires-python = ">=3.7,<4"
+requires-python = ">=3.8,<4"
 license = {file = "LICENSE"}
 authors = [{name = "ANSYS, Inc.", email = "pyansys.support@ansys.com"}]
 maintainers = [{name = "PyAnsys developers", email = "pyansys.maintainers@ansys.com"}]
```
Comment (RobPasMue) [inline pyproject.toml:13]:
```suggestion
authors = [{name = "ANSYS, Inc.", email = "pyansys.core@ansys.com"}]
maintainers =  [{name = "ANSYS, Inc.", email = "pyansys.core@ansys.com"}]
```

**Review by RobPasMue — DISMISSED** [review]

**Review by RobPasMue — APPROVED** [review]

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**